### PR TITLE
Update LICENSE.txt

### PR DESCRIPTION
**Pull request by thiagodemellobueno**
Changed the license to MIT (cleared it with Mike).

### CODE REVIEW

**Comment by thiagodemellobueno** [comment]
A'aight Pirog, she's all queuedup.

**Comment by pirog** [comment]
CRY HAVOC!!!!

**Comment by fluxsauce** [comment]
:heart_eyes_cat: Keepin' it free! @populist @patcon

File: LICENSE.txt [hunk 1]
```diff
@@ -1,42 +1,21 @@
...
-
-Governing Law, Jurisdiction and Costs
-This Agreement is governed by the laws of California, without regard to California's conflict or choice of law provisions.
-
-Severability
-If any provision of this Agreement shall be held to be invalid or unenforceable, the remainder of this Agreement shall remain in full force and effect. To the extent any express or implied restrictions are not permitted by applicable laws, these express or implied restrictions shall remain in force and effect to the maximum extent permitted by such applicable laws.
-
+The MIT License (MIT)
+
+Copyright (c) <year> <copyright holders>
```
Comment (fluxsauce) [inline LICENSE.txt:3]: You should fill this out.

**Comment by pirog** [comment]
@fluxsauce, going to do larger announcement tomorrow but wanted to provide the ole teaser first. 
@patcon, i miss your face

**Comment by pirog** [comment]
@fluxsauce just noticed that myself.

**Comment by pirog** [comment]
c5a5adc is the way

**Comment by patcon** [comment]
haha missed this. hearts and rainbows and all that junk :heart: :rainbow: :vhs: :hammer: :newspaper: :gun:

**Comment by pirog** [comment]
missed this like we all miss you @patcon  :)

ALL THE THINGS: :fallen_leaf: :ghost: :cactus: :pager:

**Pull request merged**